<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>170</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Теперь перед тобой стоит выбор: открыть игру или присоединиться к уже открытой! Не знаю, повлияет это на твой выбор или нет, но в большинстве случаев право первого хода дается создателю игры.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="create_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть свою игру</string>
   </property>
  </widget>
  <widget class="QPushButton" name="join_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Присоединиться к игре</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
